proof/refine/ARM/Syscall_R.thy

```diff
@@ -1314,14 +1360,31 @@ lemma hinv_invs'[wp]:
                         dest: st_tcb_at_idle_thread')+
   done
 
+lemma getCapReg_corres:
+  "corres (\<lambda>x y. x = to_bl y) ct_active ct_active'
+          (get_cap_reg cap_register) (getCapReg ARM_H.capRegister)"
+  apply (simp add: get_cap_reg_def getCapReg_def cap_register_def capRegister_def)
```

Comment:
Just another curisoity, but I wonder what `corressimp` could do with this one. Maybe one line it?

Reply:
Nearly! There are a few missing lemmas and so it gets stuck.